projects/bootloader/CMakeLists.txt

```diff
@@ -185,10 +115,7 @@ target_link_options(${EXECUTABLE} PRIVATE
 
 add_custom_command(TARGET ${EXECUTABLE}
     POST_BUILD
-    COMMAND ${CMAKE_OBJCOPY} -O srec --srec-len=64 ${EXECUTABLE} ${CMAKE_BINARY_DIR}/s19/${PROJECT_NAME}.s19
-    COMMAND ${CMAKE_OBJCOPY} -O ihex ${EXECUTABLE} ${CMAKE_BINARY_DIR}/hex/${PROJECT_NAME}.hex
-    COMMAND ${CMAKE_OBJCOPY} -O binary ${EXECUTABLE} ${CMAKE_BINARY_DIR}/bin/${PROJECT_NAME}.bin
```

Comment:
if you prefer to keep the local build folder under bootloader, instead of the top build folder that is currently in the root of the repo, let's also clean that folder up and remove it completely. (we also need to apply the same changes on the app project)

Reply:
Good point. I will completely purge the build at git root directory and will move the hal-drivers.cmake into the bootloader folder. Let's not touch the app for now, because is meant to be independent from the bootloader either way. I will adapt it's Cmakelists file in order to output build artifacts in the same fashion.

Reply:
So let's keep the build folder then, as it is used for the app for now. We can cleanup later

Reply:
I brought it back with the latest commit.